cerebrium/cortex/model_status.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: "Understanding Model Status"
+description: "Know your model's current state for efficient deployment and management"
+---
+
+Your Cortex model can be in one of several states at any given time. These states are used to indicate the current status of your model and its deployment.
+
+Here's what each state represents:
+
+- Active: The model is successfully deployed and available for use. If your model is sourced from a pre-built setup, it will immediately be in the active state.
+
+- Deleted: This status signifies that there is no deployment associated with the model, which usually means the model has been removed.
+
+- Deploying: The model is currently in the process of being deployed. During this state, your model is not available for use.
+
+- Starting: This is the initial state when the model deployment is underway, but none of the replicas are ready yet.
+
+- Cold: This state indicates that while a deployment exists, none of the model's replicas are ready to serve predictions.
```

Comment:
Maybe add here that, when a request is made, a startup time will be incurred - usually a few seconds